commons/clever-cloud-apis/cc-api.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Clever Cloud API
+position: 1
+---
+# Clever Cloud API
+
+Clever Cloud console is built on top of an API that allow you to manage your
+account and products. This article will explain how to connect to this API and
+use it.
+
+## Make a request on the API
+
+### Authentication
+
+Clever Cloud API works with an OAuth based authentication. First, you need to
+[create some tokens](https://console.clever-cloud.com/users/me/tokens) in
+the Clever Cloud console. Use these tokens to make an OAuth authentication.
+
+### API request links
+
+All the API links are referenced in a swagger documentation.
+
+ * [https://www.clever-cloud.com/doc/api/](https://www.clever-cloud.com/doc/api/)
+
+The base URL for the API is :
+
+ * `https://api.clever-cloud.com/v2/`
+
+## Logs API
+
+You have two ways to get logs of an application, you can get them via http or
+open a stream via websocket.
+
+### Http method
+
+You can request the logs of an application on this URL :
+
+ * `https://api.clever-cloud.com/v2/logs/<app_id>?limit=<limit>`
+ * `<app_id>` is the id of the app you request the logs.
+ * `<limit>` is the number of log lines you want to have.
+ * **Note :** For using HMAC authentication you need to sign your request with
+ this URL :
+ * `https://api.clever-cloud.com/v2/logs/<app_id>`
```

Comment:
No need to describe this for the logs API, from the user's point of view, it's part of the API. Regarding the QS not in the signature, put it in the `Authentication` part